spec/lib/services/account-api-service-spec.js

```diff
@@ -0,0 +1,154 @@
+
+        self.sandbox = sinon.sandbox.create();
+        self.sandbox.stub(waterline.localusers, 'find');
+        self.sandbox.stub(waterline.localusers, 'findOne');
+        self.sandbox.stub(waterline.localusers, 'create');
+        self.sandbox.stub(waterline.localusers, 'update');
+        self.sandbox.stub(waterline.localusers, 'destroy');
+
+        // Setup ACL rules that are missed during startServer
+        return Promise.all([
+            accountService.aclMethod('addUserRoles', 'admin', 'Administrator'),
+            accountService.aclMethod('addUserRoles', 'readonly', 'ReadOnly'),
+            accountService.aclMethod('addRoleParents', 'Administrator', ['ConfigureUsers']),
+            accountService.aclMethod('addRoleParents', 'ReadOnly', ['ConfigureSelf'])
+        ]);
+    });
+
+    after(function() {
+        self.sandbox.restore();
+    });
+
+    afterEach(function() {
+        self.sandbox.reset();
+    });
+
+    it('should list users', function() {
+        waterline.localusers.find.resolves([user1, user2]);
+        return accountService.listUsers()
+        .then(function(users) {
+            expect(waterline.localusers.find).to.have.been.called;
+            expect(users.length).to.equal(2);
+            return Promise.map(users, function(user) {
+                expect(user.username).to.be.a('string');
+                expect(user.password).to.be.an('undefined');
+                expect(user.role).to.be.a('string');
+                expect(user.privileges).to.be.instanceof(Array);
+            });
+        });
+    });
+
+    it('should get a user by name', function() {
+        waterline.localusers.findOne.resolves(user1);
+        return accountService.getUserByName('admin')
+        .then(function(user) {
+            expect(waterline.localusers.findOne).to.have.been.called;
+            expect(user.username).to.equal('admin');
+            expect(user.password).to.be.an('undefined');
+            expect(user.role).to.equal('Administrator');
+            expect(user.privileges).to.be.instanceof(Array);
+            expect(user.privileges[0]).to.equal('ConfigureUsers');
+        });
+    });
+
+    it('should create a user', function() {
+        waterline.localusers.findOne
+            .onFirstCall().resolves()
+            .onSecondCall().resolves(user1);
+        waterline.localusers.create.resolves(user1);
+        return accountService.createUser(user1)
+        .then(function(user) {
+            expect(waterline.localusers.findOne).to.have.been.called;
+            expect(waterline.localusers.create).to.have.been.called;
+            expect(user.username).to.equal('admin');
+            expect(user.password).to.be.an('undefined');
+            expect(user.role).to.equal('Administrator');
+            expect(user.privileges).to.be.instanceof(Array);
+            expect(user.privileges[0]).to.equal('ConfigureUsers');
+        });
+    });
+
+    it('should recreate a user', function() {
```

Comment:
The test case title doesn't match with the implementation, `should not recreate a duplicated user`?